### PR TITLE
add chainflag to zeroex api_fills - alt chains

### PR DESCRIPTION
**Pull request by RantumBits**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by RantumBits** [comment]
not clear on the error here - can you clarify if changes needed before review @jeff-dude ?

**Comment by RantumBits** [comment]
> not clear on the error here - can you clarify if changes needed before review @jeff-dude ?

nevermind - error cleared after merging latest commit

File: models/zeroex/arbitrum/zeroex_arbitrum_api_fills.sql [hunk 1]
```diff
@@ -309,7 +309,8 @@ SELECT
              ELSE COALESCE((all_tx.maker_token_amount_raw / pow(10, mp.decimals)) * mp.price, (all_tx.taker_token_amount_raw / pow(10, tp.decimals)) * tp.price)
              END AS volume_usd,
         tx.from AS tx_from,
-        tx.to AS tx_to
+        tx.to AS tx_to,
+         'arbitrum' AS blockchain
```
Comment (jeff-dude) [inline models/zeroex/arbitrum/zeroex_arbitrum_api_fills.sql:313]: minor point, but can you add `blockchain` column to each blockchain schema file?
i.e. arbitrum here:
https://github.com/duneanalytics/spellbook/blob/main/models/zeroex/arbitrum/zeroex_arbitrum_schema.yml

keep in mind, the deduped spell has the column already below it and uses `&` jinja syntax, meaning that should likely be moved to `api_fills` section above, so then you can use `*` syntax in the second spell.

hope that makes sense!

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
> > not clear on the error here - can you clarify if changes needed before review @jeff-dude ?
> 
> nevermind - error cleared after merging latest commit

as for this error, honestly i'm not sure what is happening. it comes up recently every so often, i think it may be when the backend cluster is overwhelmed with multiple PRs running spells concurrently

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
it's a bit compute-intensive to see all of these succeed in the CI test under an hour. i've looked closely at any potential downstream impact to this change, but i think we're fine. the only spell downstream is the deduped ones & they each hardcode blockchain again, rather than pull from here. that will work. i don't think you need to go in and update to pull from these edited spells instead.

**Pull request merged**